src/metabase/api/card.clj

```diff
@@ -639,7 +639,7 @@
      :parameters  (json/parse-string parameters keyword)
      :constraints nil
      :context     (dataset-api/export-format->context export-format)
-     :middleware  {:skip-results-metadata? true})))
+     :middleware  {:skip-results-metadata? true :format-rows? false})))
```

Comment:
Can you just change indentation to our coding standard: each keyval in it's own row, values vertically aligned:
```
{:skip-results-metadata? true 
 :format-rows?           false}
```

Reply:
Sure